src/components/footer/Footer.js

```diff
@@ -1,22 +1,62 @@
 import React from 'react';
-import { translate } from 'react-i18next';
+import { translate, Interpolate } from 'react-i18next';
 import 'i18n/i18n';
+import Panel from 'components/panel/Panel';
+import Button from 'components/button/Button';
 import styles from './footer.scss';
+import constant from 'config/constants';
 
-const Footer = ({ t }) => (
-  <footer className={styles.footerMain}>
-    <div className={styles.footerInner}>
-      <a href="#terms" className={styles.footerLink}>
-        {t('footer-terms')}
+const Footer = ({ t }) => {
+  const createTrigger = (type) => {
+    return (
+      <a href={'#' + type} className={styles.footerLink}>
+        {t(`footer-${type}`)}
       </a>
-      <a href="#cookies" className={styles.footerLink}>
-        {t('footer-cookies')}
-      </a>
-      <a href="#privacy" className={styles.footerLink}>
-        {t('footer-privacy')}
-      </a>
-    </div>
-  </footer>
-);
+    );
+  };
+
+  const doubleLineBreak = (
+    <React.Fragment>
+      <br /> <br />{' '}
```

Comment:
Was the space intentional?

```suggestion
      <br /> <br />
```